go.mod

```diff
@@ -27,28 +26,25 @@ require (
 	github.com/hashicorp/golang-lru v0.0.0-00010101000000-000000000000 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
-	github.com/mailru/easyjson v0.0.0-20190221075403-6243d8e04c3f // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/onsi/ginkgo v1.12.2
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/gomega v1.7.0
 	github.com/operator-framework/operator-sdk v0.8.0
 	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90 // indirect
-	github.com/prometheus/common v0.2.0 // indirect
-	github.com/prometheus/procfs v0.0.0-20190225181712-6ed1f7e10411 // indirect
 	github.com/sergi/go-diff v1.1.0 // indirect
-	github.com/sirupsen/logrus v1.3.0
-	github.com/spf13/pflag v1.0.3
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/pflag v1.0.5
```

Comment:
Why are all these versions changed?
I can see that the commit message says "update go.mod" but it lacks the most important information: why

Reply:
removed the commit.

Reply:
removed the commit. I saw that go.mod file changed locally. This probably happened because I switched branches locally.